testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -490,3 +495,175 @@ fn simple_neon_integration() {
 
     run_loop_thread.join().unwrap();
 }
+
+#[test]
+#[ignore]
+/// This test spins up a nakamoto-neon node.
+/// It starts in Epoch 2.0, mines with `neon_node` to Epoch 3.0, and then switches
+///  to Nakamoto operation (activating pox-4 by submitting a stack-stx tx). The BootLoop
+///  struct handles the epoch-2/3 tear-down and spin-up.
+/// This test makes three assertions:
+///  * 10 tenures are mined after 3.0 starts
+///  * Each tenure has 2 blocks
```

Comment:
Can you boost this to something like 10 blocks?

Reply:
Yep, updated in 3cef3b4.